pkgs/development/libraries/jemalloc/default.nix

```
@@ -19,6 +20,14 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-LbgtHnEZ3z5xt2QCGbbf6EeJvAU3mDw7esT3GJrs/qo=";
   };
 
+  patches = [
+    # fix tests under --with-jemalloc-prefix=, see https://github.com/jemalloc/jemalloc/pull/2340
+    (fetchpatch {
+      url = "https://github.com/jemalloc/jemalloc/commit/d00ecee6a8dfa90afcb1bbc0858985c17bef6559.patch";
+      sha256 = "sha256-11wp6byrhxi8ymqgknb40xx5xcwyj60ydajqh2029qc924ivi61p";
```

```suggestion
      hash = "sha256-11wp6byrhxi8ymqgknb40xx5xcwyj60ydajqh2029qc924ivi61p";
```

or

```suggestion
      sha256 = "11wp6byrhxi8ymqgknb40xx5xcwyj60ydajqh2029qc924ivi61p";
```

Not something in between please.